examples/docs/en-US/image.md

```diff
@@ -114,6 +114,7 @@ Besides the native features of img, support lazy load, custom placeholder and lo
 | src | Image source, same as native | string | — | - |
 | fit | Indicate how the image should be resized to fit its container, same as [object-fit](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) | fill / contain / cover / none / scale-down | — | - |
 | alt | Native alt | string | - | - |
+| referrerPolicy | Native referrerPolicy | string | - | - |
```

Comment:
we recommend using kebabcase, the same below.

Reply:
I just saw that you have `npm run lint` to fix this kind of issues.